romanisim/parameters.py

```diff
@@ -26,6 +26,16 @@
                 },
 }
 
+reference_file_names = {
+    "dark": None,
+    "distortion": None,
+    "flat": None,
+    "gain": None,
+    "inverselinearity": None,
+    "linearity": None,
+    "readnoise": None,
+    "saturation": None
+}
```

Comment:
Since we're allowing this to be either a file name or a constant, maybe reference_data?  If we moved some of the defaults in here, would that simplify the logic in image.py?  i.e.,
reference_data = { 'dark': galsim.roman.dark_current, 'flat': 1, gain: 1 u.electron / u.s, ... }

Reply:
I changed the name of the module as suggested. However, I am resistant to change the default values - I like using `None` as it doesn't "type hint," which a value would do.